src/SFA.DAS.Commitments.Support.SubSite/Mappers/ApprenticeshipMapper.cs

```diff
@@ -0,0 +1,126 @@
+
+            return new ApprenticeshipViewModel
+            {
+                FirstName = apprenticeship.FirstName,
+                LastName = apprenticeship.LastName,
+                AgreementStatus = apprenticeship.AgreementStatus.GetEnumDescription(),
+                PaymentStatus = MapPaymentStatus(apprenticeship.PaymentStatus, apprenticeship.StartDate, apprenticeship.StopDate),
+                Alerts = MapRecordStatus(apprenticeship.UpdateOriginator, apprenticeship.DataLockCourseTriaged, changeRequested),
+                ULN = apprenticeship.ULN,
+                DateOfBirth = apprenticeship.DateOfBirth,
+                CohortReference = _hashingService.HashValue(apprenticeship.CommitmentId),
+                EmployerReference = apprenticeship.EmployerRef,
+                LegalEntity = apprenticeship.LegalEntityName,
+                TrainingProvider = apprenticeship.TrainingName,
+                UKPRN = apprenticeship.ProviderId,
+                Trainingcourse = apprenticeship.TrainingName,
+                ApprenticeshipCode = apprenticeship.TrainingCode,
+                DasTrainingStartDate = apprenticeship.StartDate,
+                DasTrainingEndDate = apprenticeship.StopDate,
+                TrainingCost = apprenticeship.Cost
+            };
+        }
+
+        public ApprenticeshipSearchItemViewModel MapToApprenticeshipSearchItemViewModel(Apprenticeship apprenticeship)
+        {
+            return new ApprenticeshipSearchItemViewModel
+            {
+                HashedAccountId = _hashingService.HashValue(apprenticeship.EmployerAccountId),
+                ApprenticeshipHashId = _hashingService.HashValue(apprenticeship.Id),
+                ApprenticeName = $"{apprenticeship.FirstName} {apprenticeship.LastName}",
+                EmployerName = apprenticeship.LegalEntityName,
+                ProviderUkprn = apprenticeship.ProviderId,
+                TrainingDates = $"{apprenticeship.StartDate.ToGdsFormatWithSlashSeperator() ?? "-"} to {apprenticeship.EndDate.ToGdsFormatWithSlashSeperator() ?? "-"}",
+                PaymentStatus = MapPaymentStatus(apprenticeship.PaymentStatus, apprenticeship.StartDate, apprenticeship.StopDate),
+                DateOfBirth = apprenticeship.DateOfBirth,
+                ULN = apprenticeship.ULN
+            };
+        }
+
+        private string MapPaymentStatus(PaymentStatus paymentStatus, DateTime? startDate, DateTime? stopDate)
+        {
+            var isStartDateInFuture = startDate.HasValue && startDate.Value > new DateTime(DateTime.UtcNow.Year, DateTime.UtcNow.Month, 1);
+
+            switch (paymentStatus)
+            {
+                case PaymentStatus.PendingApproval:
+                    return "Approval needed";
+                case PaymentStatus.Active:
+                    return
+                        isStartDateInFuture ? "Waiting to start" : "Live";
+                case PaymentStatus.Paused:
+                    return "Paused";
```

Comment:
We also have a Pause date available - worth adding here?

Reply:
Good